easybuild/easyconfigs/f/fastahack/fastahack-1.0.0-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,32 @@
+# Updated: Denis Kristak (INUITS)
+easyblock = 'ConfigureMake'
+
+name = 'fastahack'
+version = '1.0.0'
+
+homepage = 'https://github.com/ekg/fastahack'
+description = """Utilities for indexing and sequence extraction from FASTA files."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+github_account = 'ekg'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-1.0.0_build-libs.patch']
+checksums = [
+    'cc1c04729b0c8ba3647cbb7e15e2b490ce701d73773f30f5892d68c36a1dceae',  # v1.0.0.tar.gz
+    '7f804486c6bafd9b1572cb5f86ff28dbebb4d6da551bde1091d6ff8f82748bf4',  # fastahack-1.0.0_build-libs.patch
+]
+
+builddependencies = [('binutils', '2.36.1')]
+
+skipsteps = ['configure']
+
+installopts = 'PREFIX=%(installdir)s '
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s', 'lib/libfastahack.%s' % SHLIB_EXT],
+    'dirs': [],
+}
+
```

Comment:
Could we add a `fastahack --help` sanity check for this one?

Reply:
@bedroge Done.